docs/source/conf.py

```diff
@@ -18,7 +24,8 @@
 
 project = "Olive"
 copyright = "2023-2025, Olive Dev team"
-version = "latest"
+version = olive.__version__
```

Comment:
I don't need this import, i want the original "latest" one. remove all unnecessary code